### ISSUE TITLE
The new block appender needs a tooltip

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#13305]
The new block appender introduced in https://github.com/WordPress/gutenberg/pull/10136 is a nice improvement. However, it misses a tooltip.

Any UI control needs to make its name visible to users. A visible name benefits all users, as it helps to clarify the meaning of controls that use only icons. For some users, a visible name is a key feature: for example, speech input users need to know the name of a control to be able to voice a proper command.

For optimal accessibility, control names should be in plain text. Gutenberg uses tooltips, which are already a trade-off. At the very least, this appender (it's a `<button>` element under the hood) needs a tooltip.
